gomock/overridable_controller_test.go

```diff
@@ -0,0 +1,34 @@
+package gomock_test
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
```

Comment:
you'll probably want to change this import path to go.uber.org/mock :)

Reply:
done